web/src/components/feature/MoleTable.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { render, screen } from '@testing-library/react'
+import { MoleTable } from './MoleTable'
+import '@testing-library/jest-dom/extend-expect'
+
```

Comment:
We should be consistent with casing. This file has a little bit of everything.

Reply:
Which parts are not consistent?